src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -291,10 +293,10 @@ private X509CertificateContainer CreateCertificate(bool isAuthority, bool isMach
                 var authorityKeyIdentifier = new AuthorityKeyIdentifier(
                     SubjectPublicKeyInfoFactory.CreateSubjectPublicKeyInfo(_authorityKeyPair.Public),
                     new GeneralNames(new GeneralName(authorityX509Name)),
-                    new BigInteger(7, _random).Abs());
+                    serialNum);
```

Comment:
Just a question -- the refactored code now uses 64-bits when before it used 7.  Is that also part of this fix?